### ISSUE TITLE
Exception At Startup

### ISSUE THREAD

**Issue by calvinnnh** [blueman-project/blueman#1001]
blueman: 2.0.5
BlueZ:
Distribution: Mint 19.1
Desktop environment: Cinnamon
* [x ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

<!--

ℹ Please use some service like [Pastebin](http://pastebin.com/) or [GitHub Gist](https://gist.github.com/) to post your logs and keep the thread clean and readable. Make sure to describe the exact steps you took when producing it.

->Receive the following exception at start-up:

[exception.docx](https://github.com/blueman-project/blueman/files/3026793/exception.docx)

**Comment by calvinnnh** [comment]
[exception.docx](https://github.com/blueman-project/blueman/files/3026797/exception.docx)

**Comment by cschramm** [comment]
The file `/var/lib/blueman/network.state` is corrupt. Please provide it so that we can look into what happend.

To get blueman running again, you can just remove the file afterward. Unless you configured a local network access point (NAP), you should be fine.

@infirit: Should we catch the `UnpicklingError` in addition to `IOError` and `UnicodeDecodeError`?

**Comment by calvinnnh** [comment]
Thanks very much for help.
[network.state.zip](https://github.com/blueman-project/blueman/files/3051456/network.state.zip)

**Comment by calvinnnh** [comment]
That solved the original problem. However; now I've lost the ability to Audio Sink my bluetooth speakers (I don't even get the option to Audio Sink). Prior to this I was getting the error message "Connection failed: blueman.bluez.errors: DbusFaledError: resource temporarily unavailable"

**Comment by calvinnnh** [comment]
I hit the wrong button> It is NOT Closed!!!

**Comment by calvinnnh** [comment]
Update. I uninstalled the pulse audio bluetooth module, then re-installed. Everything is now working correctly. Thanks for your help. It is now closed as far as I am concerned.

**Comment by calvinnnh** [comment]
Issue closed

**Comment by infirit** [comment]
@cschramm yeah adding ``UnpicklingError`` to the exceptions is a good idea.

**Comment by cschramm** [comment]
Thank you for the sample, @calvinnnh. Unfortunately I have no idea what I'm looking at there. I cannot find any pickle-ish structure in there at all... :thinking:

**Comment by calvinnnh** [comment]
@cschramm  Anything more I can do to help? I am really a novice at this stuff. I do know that when I deleted the network.state file and removed and reinstalled Pulseaudio-module-bluetooth as stated above I got everything working again. I have noticed that the little "device" icon in the tray pop-up box is no longer blue colored as in the past but functionally everything seems to be working as in the past. As I stated above before the UnpicklinError I had the "Connection failed: blueman.bluez.errors: DbusFailedError: resource temporarily unavailable" message and could not connect to my speakers even though it appeared that the device was recognized and it appeared that I was paired with the device. I have not tried removing and reinstalling blueman after implementing your suggestions even though I did try this prior to getting in touch with you, but to no avail.
